katas/content/multi_qubit_gates/index.md

```diff
@@ -140,6 +140,15 @@ The $CNOT$ gate is self-adjoint: applying it for the second time reverses its ef
     ]
 })
 
+@[exercise]({
+    "id": "multi_qubit_gates__amplitudes_swap",
```

Comment:
This task name and placement in the kata were originally my suggestion, but now I'm having second thoughts. I've forgotten entirely that all three tasks we're looking at here are strictly speaking amplitudes swap, and that we look at literally this very gate in great detail in section "Other Types of Controlled Gates" using a specific name for it... Let's move it there, before exercise "Arbitrary Controls", and call it "Anti-Controlled Gate" (task name AntiControlledGate). 

Sorry about the confusion, this more appropriate section of the kata slipped my mind completely!